docs/powergate/cli/pow_data_stage.md

```diff
@@ -0,0 +1,30 @@
+## pow data stage
+
+Temporarily stage data in the Hot layer in preparation for applying a cid storage config
```

Comment:
Nit: we've abandoned the term "layer" for "Storage", so "Hot Storage".

Reply:
cool, fixing this in powergate